packages/sit-onyx/src/components/OnyxTextarea/types.ts

```diff
@@ -22,4 +22,26 @@ export type OnyxTextareaProps = DensityProp &
     | "withCounter"
     | "message"
     | "skeleton"
-  >;
+  > & {
+    /**
+     * Override the default autosize behavior (height adjusts based on the current value).
+     * By default, the textarea will autosize while maintaining at least 3 and at most 6 rows.
+     * If the user resizes the textarea manually, the autosize will no longer work and the height
+     * set by the user is used.
+     */
+    autosize?: TextareaAutosize;
```

Comment:
IMHO resizing and autosizing should be opt-in and be disabled by default.
- resizing: Currently there is no option for the Dev to disable resizing without overwriting our internal CSS
- autosizing: API wise it seems unintuitive to have to set `min` and `max` to the same number to disable it. Additionally it would probably nice to have a prop for setting a static height based on lines. E.g.:


```suggestion
    /**
     * Enable vertical resizing of the textarea.
     */
    resizable?: boolean;
    /**
     * Set the textarea height based on linecount or enable autosizing by setting `min` or `max`.
     * By default, the textarea will autosize while maintaining at least 3 and at most 6 rows.
     * If the user resizes the textarea manually, the autosize will no longer work and the height
     * set by the user is used.
     */
    height?: number | TextareaAutosize;
```